testdata/jsight_0.3/ser-to-openapi/req.jschema.rules.type.object_03/type.object.jst

```diff
@@ -0,0 +1,11 @@
+JSIGHT 0.3
+
```

Comment:
Let's make another test for testing deep cross-structures between arrays and objects. I propose the following case:

```
{
  "object-in-object": {
    "object-in-object": {
      "foo": "bar"
    }
  },
  "short-array-in-object": [
    [ // short-array-in-short-array
      { // object-in-short-array
        "foo": "bar"
      }
    ]
  ],
  "long-array-in-object": [
    "foo",
    { // object-in-long-array
      "foo": "bar"
    },
    [ // short-array-in-long-array
      { // object-in-short-array
        "foo": "bar"
      }
    ],
    [ // long-array-in-long-array
      "foo": "bar",
      { // object-in-long-array
        "foo": "bar",
        "object-in-object": {
          "foo": "bar"
        }
      }
    ]
  ]
}
```